cobbler/items/system.py

```diff
@@ -232,7 +232,7 @@
         * ``depth``: int
         * ``server``: str - Inheritable
         * ``virt_path``: str - Inheritable
-        * ``virt_type``: str - Inheritable; One of "xenpv", "xenfv", "qemu", "kvm", "vmware", "openvz"
+        * ``virt_type``: str - Inheritable; One of "qemu", "kvm", "xenpv", "xenfv", "vmware", "vmwarew", "openvz" or "auto".
```

Comment:
This is a changelog. Could you please describe the change in the section for 3.4.0?